### PR TITLE
[hebao 1.2] minor improvements and cleanups -part 1

### PR DESCRIPTION
**Pull request by dong77**
I still need time to review all Stores, GuardianModule, GuardianUtil, SecurityModule, and ForwarderModule

### CODE REVIEW

**Review by kongliangzhong — APPROVED** [review]

File: packages/hebao_v1/contracts/modules/core/WalletFactory.sol [hunk 1]
```diff
@@ -40,20 +40,14 @@ contract WalletFactory
...
-        address             ensResolver;
-        ENSReverseRegistrar ensReverseRegistrar;
-    }
-
-    ControllerCache public controllerCache;
-
-    BaseENSManager public immutable ensManager;
+    BaseENSManager      private immutable ensManager;
+    address             private immutable ensResolver;
+    ENSReverseRegistrar private immutable ensReverseRegistrar;
```
Comment (Brechtpd) [inline packages/hebao_v1/contracts/modules/core/WalletFactory.sol:50]: Why `private` here?

**Review by Brechtpd — APPROVED** [review]

File: packages/hebao_v1/contracts/modules/core/WalletFactory.sol (hunk 1 above)
Comment (dong77) [reply packages/hebao_v1/contracts/modules/core/WalletFactory.sol:50]: WalletFactory.sol doesn't have any sub-class and this method will never be used by other contracts.

Comment (dong77) [reply packages/hebao_v1/contracts/modules/core/WalletFactory.sol:50]: Oh, I forgot to change the back. Was trying to compile to see if they are used by any other contracts.

Comment (dong77) [reply packages/hebao_v1/contracts/modules/core/WalletFactory.sol]: Changed to `public`.

**Review by Brechtpd — APPROVED** [review]

**Pull request merged**